app/utils.py

```diff
@@ -154,3 +154,14 @@ def purge_nulls_from_data(data):
 def get_request_page_questions():
     json_payload = get_json_from_request()
     return json_payload.get('page_questions', [])
+
+
+def validate_user_can_access_direct_award_project_or_403(user_id, project_id):
```

Comment:
This is named wrong - it sometimes throws a 404 :)

But also I'm not sure I like this being in 'utils' - can we just have a private method in `views/direct_award.py`?

In light of the other conversation, are we actually going to leave it in? Do we really want to do access control on POST/PUT, but _not_ on GET? That seems weird too. Hrmmm. (cc @allait )